src/main/java/org/saltyrtc/client/helpers/RandomHelper.java

```diff
@@ -13,14 +13,17 @@
 
 public class RandomHelper {
 
+    private static Random random = new Random();
+    private static SecureRandom secureRandom = new SecureRandom();
```

Comment:
Note: Re-using the RNG *might* have security implications, but it should be safe in this case as far as I can tell after some research.

Reply:
So, should I keep it as before or is this change ok?

Reply:
I think it's OK. It might improve performance slightly depending on the platform.